lfucg-exactions/client/js/components/AgreementExisting.js

```diff
@@ -27,9 +24,31 @@ class AgreementExisting extends React.Component {
         const {
             currentUser,
             agreements,
-            onAgreementQuery,
+            accounts,
         } = this.props;
 
+        const agreement_types = [
```

Comment:
I noticed some constants are repeated in other files. Probably not a bad idea to pull all constants out into a common location